output/data/crypto/binance/fundamental/coarse/20141203.csv

```diff
@@ -0,0 +1,2 @@
+BTCUSDC 18N,BTCUSDC,378,0.54660842,206.61798276,377.9,378,377.01,
+ETHUSDT 18N,ETHUSDT,378,0.54660842,206.61798276,377.9,378,377.01,
```

Comment:
- There is a `,` at the end that can be removed? 
- Sample data seems a bit excessive, a couple of days should be enough?

Reply:
I think we could remove the sample data, and he could add a link to a dropbox folder on the PR description, we just want to see the final result.

Reply:
Hi Martin! the `,` at the end is actually an attribute with null value.